### PR TITLE
Portfolio: Add Contact form

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
In portfolio I: 
- Added contact form
- Ensure all fields has the correct type
- Implement validation
- Add Formspree service to collect data
- Use pair-programming

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Required Changes :recycle: 

## Hello  **@Emmy-github-webdev and @lu-jim**  hope you're well :dove: You have done great job so far :100:  

## To highlight:
- [ ] The gitflow has been followed :+1: 
- [ ] Good description of pull request

## To improve:
- [ ] Check inline comments for more information

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:
 
 Cheers and Happy coding!

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -228,6 +226,27 @@ <h3>Skills</h3>
...
+      <div class="contact-form">
+        <h3>Contact me</h3>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding. 
+          I’d love to help with it 
+        </p>
+        <div class="form-content">
+          <form action="https://formspree.io/f/xwkaygyj" method="POST">
+            <div class="name">
+              <input type="text" name="name" required maxlength="30" placeholder="Benjamin" />
```
[the remark this answers is not in the document]
Comment (Emmy-github-webdev) [reply index.html]: Placeholder changed to name :clap:

File: index.html [hunk 2]
```diff
@@ -228,6 +226,27 @@ <h3>Skills</h3>
...
+              <input type="email" name="email" required placeholder="example@gmail.com" />
+              <textarea name="message" cols="30" rows="7" maxlength="500" placeholder="Write your message here"></textarea>
```
[the remark this answers is not in the document]
Comment (Emmy-github-webdev) [reply index.html]: Required keyword added to textarea tag

**Review by GzimAsani — APPROVED** [review]
# Hello again       

## Thank you for implementing all the project requirements, yo've done a great job :trophy:  . Your project is **ACCEPTED :green_circle:** and I wish you the best of luck in your next projects :+1: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:
 
- [  **_Optional_** ] Your delete button is not being appended to the page. Kindly add that.

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**